### DISCUSSION TITLE
console command usage

### DISCUSSION THREAD

**Discussion by ggw-** [ipxe/ipxe#1178] [General]
IGNORE THIS - LAYER 8 ISSUE.

 I compiled iPXE from a git pull a few days ago.    I've used the "console" command before to display a background image but this round I'm having some problems doing so.
 
 Its saying during boot,  "**console: command not found**".
 
 I checked general.h;  "#define CONFIG_CMD",  "IMAGE_PNG" and in console.h "CONSOLE_FRAMEBUFFER" are all defined.   
 
I'm using "_make bin-x86_64-efi/ipxe.efi EMBED=chain.ipxe_" to build the binary.    Then copying ipxe.efi to /tftpboot/ replacing the the one there.   
   
Command line in the boot file is:
_console --x 1024 --y 768 --picture http://mirror.workplace.com/ipxe/ipxebg1.png_

If I comment out that line,  the boot process continues as expected.

Any idea what I may be missing?  

Thanks in advance for any comments/suggestions.

![ipxe_console_error1](https://github.com/ipxe/ipxe/assets/10251349/919a4cd3-e096-4c6e-8f97-b751768a7621)

**Comment by NiKiZe** [comment]
console command is not defined..
You said that you have #define CONFIG_CMD in general.h, are you sure?, And is the version you built with this really the file on the server?

I would put these in config/local/general.h as to not cause any conflicts with future updates https://ipxe.org/buildcfg/console_cmd

**Comment by ggw-** [comment]
NiKZe - just realized what i did to myself,  i pulled the repo overwriting the files.   I'll use the config/local/ method going forward,  thanks for that useful bit of info too,  appreciate it.  
thank you!
-greg
